app/converter.py

```diff
@@ -0,0 +1,33 @@
+from json import load, dump
+
+from app.rules.v2_to_v3.root_conversions import general_conversions
+from app.rules.v2_to_v3.block_conversions import block_conversions
+from app.rules.v2_to_v3.question_conversions import question_conversions
+from app.rules.v2_to_v3.answer_conversions import answer_conversions
+from app.rules.v2_to_v3.group_conversions import group_conversions
+from app.rules.v2_to_v3.routing_rules import routing_conversions
+from app.helpers.parse_schema import QuestionnaireSchema
+
+
+class Converter:
+    def __init__(self, file_name):
+        with open("schemas/to_convert/" + file_name, encoding="utf8") as schema_data:
```

Comment:
```suggestion
        with open("schemas/to_convert/" + file_name, 'r', encoding="utf8") as file:
```
The context manager is a file handler not the actual data